gce/dns-records.tf

```diff
@@ -45,3 +45,11 @@ resource "google_dns_record_set" "docker-registry" {
   ttl = "60"
   rrdatas = ["${google_compute_instance.docker-registry.network_interface.0.address}"]
 }
+
+resource "google_dns_record_set" "postgresapi" {
+  managed_zone = "${var.dns_zone_id}"
+  name = "${var.env}-postgresapi.${var.dns_zone_name}"
+  type = "A"
```

Comment:
see https://github.com/alphagov/tsuru-terraform/pull/66 - it's being solved there